packages/palette_generator/README.md

```diff
@@ -12,11 +13,5 @@ To use this package, add `palette_generator` as a
 
 ## Example
 
-Import the library via
-
-```dart
-import 'package:palette_generator/palette_generator.dart';
-```
-
 Then use the `PaletteGenerator` Dart class in your code. To see how this is done,
```

Comment:
"Then" no longer makes much sense here, now that the previous lines have been removed. How about "```To see how to use the `PaletteGenerator` class in your code, see [the example app](https://pub.dev/packages/palette_generator/example).```"